packages/manager/apps/pci-load-balancer/src/api/hook/useL7Policy.tsx

```diff
@@ -118,3 +122,36 @@ export const useDeletePolicy = ({
     ...mutation,
   };
 };
+
+type CreatePolicyProps = {
+  projectId: string;
+  listenerId: string;
+  region: string;
+  policy: TL7Policy;
+  onError: (cause: Error) => void;
+  onSuccess: (newPolicy: TL7Policy) => void;
+};
+
+export const useCreatePolicy = ({
+  projectId,
+  listenerId,
+  policy,
+  region,
+  onError,
+  onSuccess,
+}: CreatePolicyProps) => {
+  const mutation = useMutation<TL7Policy>({
+    mutationFn: async () => createPolicy(projectId, region, listenerId, policy),
+    onError,
+    onSuccess: async (newPolicy) => {
+      await queryClient.invalidateQueries({
+        queryKey: ['l7Policies'],
+      });
+      onSuccess(newPolicy);
+    },
+  });
+  return {
+    createPolicy: () => mutation.mutate(),
```

Comment:
The method exported from `packages/manager/apps/pci-load-balancer/src/api/data/l7Policies.ts`has the same exact name, may be name it something else, "doCreatePolicy" maybe?